src/kili/services/export/format/kili/__init__.py

```diff
@@ -43,6 +45,14 @@ def _save_assets_export(self, assets: List[Asset], output_filename: Path):
                     with (labels_folder / f"{external_id}.json").open("wb") as output_file:
                         output_file.write(asset_json.encode("utf-8"))
             self.create_readme_kili_file(root_folder)
+
+            if self.download_media:
+                images_folder = base_folder / "images"
+                images_folder.mkdir(parents=True)
+                for asset in assets:
+                    if os.path.isfile(asset["content"]):
+                        shutil.move(asset["content"], images_folder)
+
```

Comment:
can't you find a way to define the base folder before to direclty download the assets in the right folder when calling assets ?